README.md

```diff
@@ -79,124 +78,63 @@ The application will now be available from your browser at http://localhost:8081
-| `helm install`               | :heavy_check_mark:       |
-
-#### Hosted
-
-TBD
-
-## Installing the plugin
-
-There are a range of options for installing the Helm plugin. You'll need to build it first, and
-then install the plugin with the options shown below:
-
-### Temporary Install
-
-Installations done via the Karaf console will be wiped out with every restart of Nexus Repository. This is a
-good installation path if you are just testing or doing development on the plugin.
-
-* Enable Nexus console: edit `<nexus_dir>/bin/nexus.vmoptions` and change `karaf.startLocalConsole`  to `true`.
+### Permanent Reinstall
 
-  More details here: [Bundle Development](https://help.sonatype.com/display/NXRM3/Bundle+Development+Overview)
-
-* Run Nexus' console:
-  ```
-  # sudo su - nexus
-  $ cd <nexus_dir>/bin
-  $ ./nexus run
-  > bundle:install file:///tmp/nexus-repository-helm-0.0.12.jar
-  > bundle:list
-  ```
-  (look for org.sonatype.nexus.plugins:nexus-repository-helm ID, should be the last one)
-  ```
-  > bundle:start <org.sonatype.nexus.plugins:nexus-repository-helm ID>
-  ```
-
-### (more) Permanent Install
-
-For more permanent installs of the nexus-repository-helm plugin, follow these instructions:
-
-* Copy the bundle (nexus-repository-helm-0.0.12.jar) into <nexus_dir>/deploy
-
-This will cause the plugin to be loaded with each restart of Nexus Repository. As well, this folder is monitored
-by Nexus Repository and the plugin should load within 60 seconds of being copied there if Nexus Repository
-is running. You will still need to start the bundle using the karaf commands mentioned in the temporary install.
-
-### (most) Permanent Install
-
-If you are trying to use the Helm plugin permanently, it likely makes more sense to do the following:
-
-* Copy the bundle into `<nexus_dir>/system/org/sonatype/nexus/plugins/nexus-repository-helm/0.0.12/nexus-repository-helm-0.0.12.jar`
-* Make the following additions marked with + to `<nexus_dir>/system/org/sonatype/nexus/assemblies/nexus-core-feature/3.x.y/nexus-core-feature-3.x.y-features.xml`
+* Copy the new bundle into `<nexus_dir>/system/org/sonatype/nexus/plugins/nexus-repository-helm/<helm_version>/nexus-repository-helm-<helm_version>.jar`
+* If you are using OSS edition, make these mods in: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-oss-feature/3.x.y/nexus-oss-feature-3.x.y-features.xml`
+* If you are using PRO edition, make these mods in: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-pro-feature/3.x.y/nexus-pro-feature-3.x.y-features.xml`
 
    ```
-         <feature prerequisite="false" dependency="false">nexus-repository-rubygems</feature>
-   +     <feature prerequisite="false" dependency="false">nexus-repository-helm</feature>
-         <feature prerequisite="false" dependency="false">nexus-repository-gitlfs</feature>
+         <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-rubygems</feature>
```

Comment:
Trivial suggestion: While just a placeholder, suggest x.y.z rather than x.y.xy, since xy looks like some combination of x and y which is untrue=)